environments/environment.properties

```diff
@@ -0,0 +1,5 @@
+mvn.environment.name=${env.BUILD_ENV:dev}
+default.context.name=default
+docker.image.search=docker.elastic.co/elasticsearch/elasticsearch:7.9.1
```

Comment:
I would change our docker image here to 
`opensearchproject/opensearch:1`

Reply:
Yep you are correct,  now I have green test I will change this there were too many factors affecting the tests from what is on master.